distribution/sdk/sample-cometd/cometd-web-example/src/main/webapp/js/application.js

```diff
@@ -65,7 +65,7 @@
             // Add Notification Subscription
             cometd.subscribe('/ddf/notifications/**', function(message) {
               console.log("Got a message!");
```

Comment:
I don't think it's necessary. It's in the sdk because it's showing what's going on behind the scenes. If an integrator puts console logs in their code, they're probably doing it for a reason or they're blindly copying our sdk code. In either case, a comment won't prevent that